paper/paper.md

```diff
@@ -108,7 +108,7 @@ Additionally, if a record of stimulus or trigger times is provided `pyNeuroTrace
 ![Average stimulus-locked responses from the same *in vivo* imaging experiment of a dendritic branch in {fig:heatmap} \label{fig:AvgResponse}](docs/img/pyn_avg_response_on_off_9.png)
 
 # GPU Acceleration
-Several of the filters in `pyNeuroTrace` have been rewritten to be almost entirely vectorized in their calculations. The benefit is more noticeable when comparing the difference in performance while using large data sets, such as those generated using a longer time series or faster acquisition rates. These vectorized implementations gain further speed by being executed on a GPU using the Cupy Python library [@cupy_learningsys2017]. The GPU-accelerated filters can be imported from the `pyneurotrace.gpu.filters` module, and a CUDA-compatible graphics card is required for their execution. This functionality is becoming increasingly crucial as acquisition rates increase for kilohertz imaging of activity, which can generate arrays of hundreds of thousands of data points in just a few minutes of recording. \autoref{fig:CPUvsGPU} shows the difference in calculating arrays of various sizes using either the CPU or vectorized GPU-based approach of the $\Delta F/F$ function. The CPU used in these calculations was an Intel i5-9600K with six 4.600GHz cores; the GPU was an NVIDIA GeForce RTX 4070 with CUDA Version 12.3. 
+Several of the filters in `pyNeuroTrace` have been rewritten to be almost entirely vectorized in their calculations. The benefit is more noticeable when comparing the difference in performance while using large data sets, such as those generated using a longer time series or faster acquisition rates. These vectorized implementations gain further speed by being executed on a GPU using the Cupy Python library [@cupy_learningsys2017]. The GPU-accelerated filters can be imported from the `pyneurotrace.gpu.filters` module, and a CUDA-compatible graphics card is required for their execution. This functionality is becoming increasingly crucial as acquisition rates increase for kilohertz imaging of activity [@Zhang2019], which can generate arrays of hundreds of thousands of data points in just a few minutes of recording. \autoref{fig:CPUvsGPU} shows the difference in calculating arrays of various sizes using either the CPU or vectorized GPU-based approach of the $\Delta F/F$ function. The CPU used in these calculations was an Intel i5-9600K with six 4.600GHz cores; the GPU was an NVIDIA GeForce RTX 4070 with CUDA Version 12.3. When compared, the GPU calculation was on average TODO times faster on time series up to 1 million points in size.
```

Comment:
@Peter-Hogg I left this as `TODO`, once merged are you able to replace it with the actual average speed difference? visually looks like 100 or so.

Reply:
the mean was 103.5 times faster when comparing between the two lines, the max is about 150 and that's for 100,000 not 1 million

Reply:
Thanks, updated to 'in the order of 100 times faster' and changed to 100,000